webapp/src/components/Celebrities.vue

```diff
@@ -204,26 +214,53 @@
         // initialize lists of boxes and markers to be drawn
         var boxMap = new Map();
         var markers = [];
-        var es_data = this.elasticsearch_data
+        var es_data = this.elasticsearch_data;
+        var i=0;
         es_data.forEach( function(record) {
```

Comment:
Not a deal breaker, but from what I understand forEach can be about 95% slower than the using a for loop like so: `for (let i=0; i<somevar; i++)`. Imo it can be a bit more readable as well, but this is really subjective. I'm not sure if there are any vue / spa specific guidelines for this, might be worth looking into.

Reply:
I've added a github issue for this. I'll make this change in the next sprint.
https://github.com/awslabs/aws-media-insights-engine/issues/88